frontend/src/components/notification/notification-dropdown.tsx

```diff
@@ -46,16 +47,18 @@ const NotificationDropdown = ({user, token}: UserProps) => {
 
         fetchUserNotification();
 
-    }, []);
+    }, [token]);
     return (
-        <div>
+        <div className="flex flex-row gap-4">
             {user.role == "admin" || user.role == "moderator" || user.role == "analyst" ? (
                 <DropdownMenu>
                     <DropdownMenuTrigger asChild>
-                        <Button variant="ghost">
-                            <BellIcon className="h-4 w-4 mr-2" />
-                            Article Queue
+                        <Button variant="ghost" className="relative">
+                            <ShieldAlert
+                             className="h-4 w-4 mr-2" />
+                            <p className="hidden xl:block">Article Queue</p>
                             {queueNotifications.length ? ` (${queueNotifications.length})` : ""}
```

Comment:
move here for spacing

```suggestion
                            <p className="hidden xl:block">Article Queue {queueNotifications.length ? `(${queueNotifications.length})` : ""}</p>
                           
```